tests/unit/dataactbroker/test_handlers_fileHandler_sign_url.py

```diff
@@ -2,31 +2,42 @@
 from dataactbroker.handlers import fileHandler
 from dataactcore.interfaces.interfaceHolder import InterfaceHolder
 from unittest.mock import Mock
-from flask import Flask
+import json
 
 def test_get_signed_url_for_submission_file_local(database, monkeypatch):
     submission = SubmissionFactory()
     database.session.add(submission)
     database.session.commit()
 
-    with Flask(__name__).test_request_context('?file=file_name&submission='+str(submission.submission_id)):
-        interfaces = InterfaceHolder()
-        file_handler = fileHandler.FileHandler(Mock(), interfaces=interfaces, isLocal=True, serverPath="/test/server/path/")
-        file_handler.check_submission_permission = Mock()
-        monkeypatch.setattr(
-            fileHandler, 'send_from_directory', Mock(return_value='send from directory reached')
-        )
-        assert file_handler.get_signed_url_for_submission_file() == 'send from directory reached'
+    interfaces = InterfaceHolder()
+    file_handler = fileHandler.FileHandler(Mock(), interfaces=interfaces, isLocal=True, serverPath="/test/server/path/")
+    file_handler.check_submission_permission = Mock()
+
+    mock_dict = Mock()
+    mock_dict.return_value.getValue.side_effect = ['file_name', str(submission.submission_id)]
+    monkeypatch.setattr(fileHandler, 'RequestDictionary', mock_dict)
+
+    json_response = file_handler.get_signed_url_for_submission_file()
+    print(json.loads(json_response.get_data().decode("utf-8")))
+    assert json.loads(json_response.get_data().decode("utf-8"))['url'] == "/test/server/path/file_name.csv"
 
 def test_get_signed_url_for_submission_file_s3(database, monkeypatch):
     submission = SubmissionFactory()
     database.session.add(submission)
     database.session.commit()
 
-    with Flask(__name__).test_request_context('?file=file_name&submission=' + str(submission.submission_id)):
-        interfaces = InterfaceHolder()
-        file_handler = fileHandler.FileHandler(Mock(), interfaces=interfaces, isLocal=False)
-        file_handler.check_submission_permission = Mock()
-        monkeypatch.setattr(fileHandler, 's3UrlHandler', Mock())
+    interfaces = InterfaceHolder()
+    file_handler = fileHandler.FileHandler(Mock(), interfaces=interfaces, isLocal=False)
+    file_handler.check_submission_permission = Mock()
+
+    mock_dict = Mock()
+    mock_dict.return_value.getValue.side_effect = ['file_name', str(submission.submission_id)]
+    monkeypatch.setattr(fileHandler, 'RequestDictionary', mock_dict)
+
+    mock_dict = Mock()
+    mock_dict.return_value.getSignedUrl.side_effect = ['/signed/url/path/file_name.csv']
```

Comment:
Little detail, but if you want this to always return just one value, you should use the more standard
```python
mock_dict.return_value.getSignedUrl.return_value = '/signed/url/path/file_name.csv'
```